src/help/components/contextual/ContextualHelpHandler.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from 'react';
+import { useRouteMatch } from 'react-router-dom';
+import { Location, LocationToPath } from '../../../app/config/urls';
+
+import './styles/contextual-help.scss';
+
+const eventHandler = (event: MouseEvent) => {
+  const element = event.target as HTMLElement;
+  if (element.dataset.articleId) {
+    console.log(element.dataset.articleId);
+  }
+};
+
+const ContextualHelpHandler = () => {
+  const isHelpResults = useRouteMatch(LocationToPath[Location.HelpResults]);
+  const isHelpEntry = useRouteMatch(LocationToPath[Location.HelpEntry]);
```

Comment:
You can skip that one, the previous one will always be truthy (but with the `exact` flag to false) if this one is